apps/web/src/components/templates/ExecutionDetailsModalWrapper.tsx

```diff
@@ -0,0 +1,42 @@
+import { useQuery } from 'react-query';
+import { getActivityList } from '../../api/activity';
+import { ExecutionDetailsModal } from '../activity/ExecutionDetailsModal';
+import { LoadingOverlay, useMantineTheme } from '@mantine/core';
+import { colors } from '../../design-system';
+
+interface Props {
+  transactionId: string;
+  isOpen: boolean;
+  onClose: () => void;
+}
+
+export const ExecutionDetailsModalWrapper = ({ transactionId, isOpen, onClose }: Props) => {
+  const theme = useMantineTheme();
+  const {
+    data: notification,
+    isLoading,
+    isFetching,
+  } = useQuery<{ data: any[] }>(['activitiesList', transactionId], () => getActivityList(0, { transactionId }), {
```

Comment:
Was not sure if using this endpoint which is originally a list is better than somehow renaming it or querying a different way